armi/interfaces.py

```diff
@@ -310,6 +317,10 @@ def interactCoupled(self, iteration):
         """Called repeatedly at each time node/subcycle when tight physics couping is active."""
         pass
 
+    def getTightCouplingValue(self):
+        """abstract method to retrive the value in which tight coupling is to converge on"""
+        pass
+
```

Comment:
Are these value(s) singular? Can you provide an example in this docstring?

Reply:
This approach may change. Let's chat offline this afternoon and do some brainstorming.